x-pack/plugins/ml/public/jobs/new_job/simple/recognize/check_module.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import chrome from 'ui/chrome';
+import { mlJobService } from '../../../../services/job_service';
+import { ml } from '../../../../services/ml_api_service';
+import { toastNotifications } from 'ui/notify';
+
+
+// Checks whether the jobs in a data recognizer module have been created.
+// Redirects to the Anomaly Explorer to view the jobs if they have been created,
+// or the recognizer job wizard for the module if not.
+export function checkViewOrCreateJobs(Private, $route, kbnBaseUrl, kbnUrl, i18n) {
```

Comment:
Can't `i18n` just be imported into the file like normal instead of passing it as an argument here?

Reply:
AFAIK originally it was done like this by the translation team. But since [this PR](https://github.com/elastic/kibana/pull/36329) is up now I agree we should just import it.

Reply:
Will change it to `import { i18n } from '@kbn/i18n';`